applications/rag/example_notebooks/rag-kaggle-ray-sql-latest.ipynb

```diff
@@ -272,6 +271,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Port forward to the Ray dashboard to see job status: kubectl port-forward -n <namespace> service/ray-cluster-kuberay-head-svc 8265:8265\n",
```

Comment:
Doesn't this depend on the IAP settings? Can we just link to the README's verify/debug section?

Reply:
The port forward should work regardless of IAP, I think.. @blackzlq ?

Reply:
yes, confirmed it works regardless